Add missing info to README (#39)
bactensor/bt-ddos-shield: README.md

metagraph = ShieldMetagraph(wallet, netuid, network)
```

> If you are running Validator in Docker, make sure to mount volume to the container for storing certificate generated for

Reviewer: ```suggestion
If you are running Validator in Docker, make sure to mount a volume for storing certificate generated for
```
here we are talking about a scenario when user first creates the keys, and the volume is required for storing the generated stuff, right?
I mean I wonder about `for storing`... because after it is stored for the first time, vali needs to mount the volume to keep using it, doesn't it? I don't have a strong opinion, but maybe more clear if we explained this here too :thinking:

Author: Suggestion applied. As for `for storing` there is link to section with more details - there is all info. I think this should be enough.

> encrypting addresses - see [Encryption key and cert](#encryption-key-and-cert) for more information. If you are mounting

Reviewer: It is a bit confusing as the encryption key and cert doesn't mention volumes at all :upside_down_face: 
I am not sure if that section would be the right place to mention again - If you run in a docker, make sure to have this files visible in a volume?

Author: Added some explanation for this

> directory with wallet to the container, you can use same volume for the certificate.

Reviewer: ```suggestion
a volume with wallet to the container, you can use the same volume for the certificate.
```

### Advanced usage:

#### Encryption key and cert

Upon first call of `ShieldMetagraph`, by default, a cert-key pair will be created, saved on disk and pushed to the Subtensor. If for
whatever reason one needs to provide their own pregenerated cert-key pair (for example when moving to a new validator node), make sure to put the
cert and key files on the server and provide `VALIDATOR_SHIELD_CERTIFICATE_PATH` env var (or use certificate_path option in options passed

> via constructor) when starting the new validator instance. Default value for this env var is `./validator_cert.pem`.

Reviewer: I think I would add one more sentence here about the pem needing to be visible in docker if this is run in a docker at the end :)

Author: Added in another place
